tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md

```diff
@@ -67,27 +58,56 @@ In addition to global accounts and subaccounts, Cloud Foundry includes another h
 ![Cloud Foundry trial account model](cf-trial-account-model.png)
 
 [VALIDATE_2]
-
 [ACCORDION-END]
+[ACCORDION-BEGIN [Step : ](Navigation in the cockpit)]
 
-[ACCORDION-BEGIN [Step 3: ](Create a Cloud Foundry trial account)]
-As previously mentioned, a Cloud Foundry trial account is not automatically created when you register for a trial on SAP Cloud Platform. To create one, you must do the following:
+There are 2 ways of navigating in the cockpit:
 
-1. Click on the **Cloud Foundry Trial** tile on the [cockpit homepage](https://cockpit.hanatrial.ondemand.com/cockpit/#/home/trialhome).
+- using the left hand-side navigation
+- using the breadcrumbs at the top of the page
 
-    ![Environments on SAP Cloud Platform trial](trial-tiles.png)
+There are certain actions you can perform at each level - we will refer to the sum of these actions as a **scope**. For example, when you are in a global account you can add global account members, create subaccounts and add entitlements to subaccounts. They represent the **global account scope**.
```

Comment:
Here I would add back the sentence before:
"Remember all that stuff about global accounts, subaccounts we talked about earlier? " 
Otherwise, on it's own it's not clear what "level" we're talking about. Or it can be rephrased a bit to make it clear, something small like:

There are certain actions you can perform at each hierarchical level (global account, subaccount, space etc.) - we will refer to the sum of these actions as a **scope**.